core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.scss

```diff
@@ -24,6 +24,7 @@ dot-whats-changed,
     max-width: 100%;
     padding: $spacing-5;
     width: 100%;
+    overflow: scroll;
```

Comment:
I'm not sure, but maybe this draws and empty scroll bar when there's no overflow. Maybe setting it to `auto` will work better